quesma/quesma/schema_transformer.go

```diff
@@ -398,6 +398,7 @@ func (s *SchemaCheckPass) applyFullTextField(query *model.Query) (*model.Query,
 		return query, nil
 	}
 	fullTextFields := table.GetFulltextFields()
+	sort.Strings(fullTextFields)
```

Comment:
I'd comment why sorting is important here, because it's not very straightforward when reading the code :) 
Also it's not something we can easily test

Reply:
Too, though sorted, deterministic order is usually required for stable tests and caching.